test/service/test_manifest.py

```diff
@@ -332,83 +332,97 @@ def test_manifest_zarr(self):
             },
             {
                 'file_crc32c': 'caaefa77',
                 'file_name': '377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr/expression_matrix/expression/0.0',
                 'file_uuid': '0bef5419-739c-4a2c-aedb-43754d55d51c',
+                'file_drs_uri': '/0bef5419-739c-4a2c-aedb-43754d55d51c?version=2018-10-10T031039.642846Z',
                 'specimen_from_organism_organ': 'brain'
             },
             {
                 'file_crc32c': 'f629ec34',
                 'file_name': '377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr/expression_matrix/gene_id/.zarray',
                 'file_uuid': '3a5f7299-1aa1-4060-9631-212c29b4d807',
+                'file_drs_uri': '/3a5f7299-1aa1-4060-9631-212c29b4d807?version=2018-10-10T031039.899615Z',
                 'specimen_from_organism_organ': 'brain'
             },
             {
                 'file_crc32c': '59d86b68',
                 'file_name': '377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr/expression_matrix/gene_id/0.0',
                 'file_uuid': 'a8f0dc39-6019-4fc7-899d-4e34a48d03e5',
+                'file_drs_uri': '/a8f0dc39-6019-4fc7-899d-4e34a48d03e5?version=2018-10-10T031040.113268Z',
                 'specimen_from_organism_organ': 'brain'
             },
             {
                 'file_crc32c': '25d193cf',
                 'file_name': '377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr/expression_matrix/qc_metric/.zarray',
                 'file_uuid': '68ba4711-1447-42ac-aa40-9c0e4cda1666',
+                'file_drs_uri': '/68ba4711-1447-42ac-aa40-9c0e4cda1666?version=2018-10-10T031040.583439Z',
                 'specimen_from_organism_organ': 'brain'
             },
             {
                 'file_crc32c': '17a84191',
                 'file_name': '377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr/expression_matrix/qc_metric/0.0',
                 'file_uuid': '27e66328-e337-4bcd-ba15-7893ecaf841f',
+                'file_drs_uri': '/27e66328-e337-4bcd-ba15-7893ecaf841f?version=2018-10-10T031040.801631Z',
                 'specimen_from_organism_organ': 'brain'
             },
             {
                 'file_crc32c': '25d193cf',
                 'file_name': '377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr/expression_matrix/qc_values/.zarray',
                 'file_uuid': '2ab1a516-ef36-41b6-a78f-513361658feb',
+                'file_drs_uri': '/2ab1a516-ef36-41b6-a78f-513361658feb?version=2018-10-10T031040.958708Z',
                 'specimen_from_organism_organ': 'brain'
             },
             {
                 'file_crc32c': 'bdc30523',
                 'file_name': '377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr/expression_matrix/qc_values/0.0',
                 'file_uuid': '351970aa-bc4c-405e-a274-be9e08e42e98',
+                'file_drs_uri': '/351970aa-bc4c-405e-a274-be9e08e42e98?version=2018-10-10T031041.135992Z',
                 'specimen_from_organism_organ': 'brain'
             }
         ]
-
+        for i in range(len(expected)):
```

Comment:
Since the items in the list are dicts there is no need to access them by index in order to be able to modify them.